internal/cli/atlas/streams/instance/download.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2024 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package instance
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/usage"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	atlasv2 "go.mongodb.org/atlas-sdk/v20231115008/admin"
+)
+
+var downloadMessage = "Download of %s completed.\n"
+
+type DownloadOpts struct {
+	cli.GlobalOpts
+	cli.DownloaderOpts
+	tenantName string
+	start      int64
+	end        int64
+	store      store.StreamsDownloader
+}
+
+func (opts *DownloadOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+func (opts *DownloadOpts) Run() error {
+	params := atlasv2.DownloadStreamTenantAuditLogsApiParams{
+		GroupId:    opts.ProjectID,
```

Comment:
```suggestion
		GroupId:    opts.ConfigProjectID(),
```
so it also picks the one in the config or the env variable

Reply:
Did not know about that, fixed in 488dbf4ed